templates/index.html

```diff
@@ -7,5 +7,6 @@
     <li><a href="/twitter/">Twitter Feed</a>
     <li><a href="/log/">Logs</a>
     <li><a href="/contributors/">Contributors data</a>
+    <li><a href="/openhub_organizations/">OpenHub Organizations</a>
```

Comment:
no, we only need the coala org initially.  so just `/openhub/` , and list the projects in OpenHub under the "coala" org (where coala is determined by the repo owner name).

there is some fun analytics we can do linking to other orgs, but importing 'all orgs' is too much data; it takes too long, and Netlify kills the job